3rd_party/include/transwarp/transwarp.h

```diff
@@ -0,0 +1,3064 @@
+/// @mainpage transwarp is a header-only C++ library for task concurrency
```

Comment:
3 files need adding for this in https://github.com/elastic/ml-cpp/tree/master/3rd_party/licenses:

1. transwarp-INFO.csv (with the following columns: name,version,revision,url,license,copyright)
2. transwarp-NOTICE.txt (zero length file)
3. transwarp-LICENSE.txt (plain text copy of https://github.com/bloomen/transwarp/blob/master/LICENSE.md)